goalgorilla/open_social
# Issue #3102157 by rolki: Add the ability to switch teaser style in featured content block

## Problem
As a CM, I want to switch teaser style in a featured content block

## Solution
Create a new ability to change the layout for a featured content block.

## Issue tracker
https://www.drupal.org/project/social/issues/3102157
https://getopensocial.atlassian.net/browse/YANG-1492

## How to test
* Create a new landing page
* Add section "Featured Content"
* You should see at the bottom of section "Layout" select list and you have to choose "Vertical" mode
* Save the page
* You should see wide teasers instead of default view mode

## Screenshots
![c9787faa-c64d-4c4e-8769-782fc4b40e38](https://user-images.githubusercontent.com/50984627/73647990-4b87b100-4685-11ea-853a-24fab97cd4a9.png)

## Release notes
<describe the release notes>

## Change Record


## Translations
